lib/Text/Xslate/Runner.pm

```diff
@@ -212,12 +213,14 @@ sub getopt_spec { @Spec }
 sub _build_getopt_spec {
     my($self) = @_;
 
+    $DB::single = 1;
+    
     my @spec;
     foreach my $attr($self->meta->get_all_attributes) {
```

Comment:
aha, this would be a contributor to the slowdown -- this Moo class is being upgraded to Moo to get the list of attributes. (thanks @shadowcat_mst)